usagov/usagov-2021
# Usagov 1386 flag cards style update

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1386

## Description

For consistency across the styles of the site, let's restyle the flag cards following the following:
* Remove the aqua underline from the regular state of the card
* Remove the drop shadow from the over state of the card
* Add an aqua underline to the hover state of the card
* Add the correct focus state: The inner stroke is 2px #FFFFFF. The outer stroke is 2px #0186a2.
* Keep the regular photo for the hover state of the card (versus the current color change photo) __updated__ added in the focus state to this ticket

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Use screenshots to compare against test validation
<img width="1512" alt="Screenshot 2024-02-29 at 3 04 48 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/ed42bb9f-dc35-4e4c-89e5-ce5d2be3bfea">

<img width="1512" alt="Screenshot 2024-02-29 at 3 04 53 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/1358767d-200f-4d24-93a8-e28f19f05c26">

### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/housing-voucher-section-8 and look at the flag card at the bottom of the page
- Hover over the card and verify the blue underline shows up under the link title and that there is no box shadow
- Tab to the flag card and verify that on focus the card image doesn't get brighter, the focus color is #0186a2 and that there is no box shadow

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
